neuralmonkey/tests/test_vocabulary.py

```diff
@@ -49,6 +49,7 @@ def test_there_and_back_self(self):
     def test_min_freq(self):
 
         vocabulary = Vocabulary()
+        vocabulary.correct_counts = True
```

Comment:
tohle ale nespadne protoze jsem tam pridal flag ze kdyz pridavat tokenized text tak to automaticky zapne correct_counts ... takze defakto bych mohl odstranit ten radek: vocabulary.correct_counts = True